smart_campus/app/templates/app/reward_list_page.html

```diff
@@ -0,0 +1,52 @@
+{% extends "app/base.html" %}
+{% load static %}
+
+{% block title %}Smart Campus{% endblock %}
+
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static " app/css/list.css " %}">
+{% endblock %}
+
+{% block content %}
+  <div class="ui borderless center aligned text container">
+    {% if messages %}
+      {% for message in messages %}
+    <div class="ui info message">
+      {{ message }}
+    </div>
+      {% endfor %}
+    {% endif %}
+  {% if rewards %}
+    {% for reward in rewards %}
+    <div class="ui cards">
+      <div class="ui centered card">
+        <div class="image">
+            <img src="{{ reward.image.url }}">
+        </div>
+        <div class="content">
+          <!-- <img class="right floated tiny ui image" src="{{ reward.image }}"> -->
```

Comment:
**DO NOT** version control **ANY** code that is commented out.

Reply:
okay, removed it.